examples/spring-boot-draft/README.md

```diff
@@ -0,0 +1,107 @@
+SpringBoot Demo for using Jib
+
+Tips: this is just a draft, please contribute if you have good suggestion.
+
+### Quickstart
+
+1.Modify configuration:
+
+Modify `CUSTOM_REGISTRY_URL`, `YOUR_USER_NAME`, `YOUR_PASSWORD`, `PASSWORD_ENCRYPT_BY_MAVEN` as yourself, and change `credHelper` to match your platform .
+
+2.Build the image:
+
+Run `gradle jib` or `mvn compile jib:build` under project dictionary .
+
+3.Run container:
+
+Run `docker run --name hellojib -p 8080:8080 YOUR_USER_NAME/hellojib:jib` .
+
+4.Access application:
+
+Run `curl localhost:8080` and it will return `"Hello Jib"`, run container success .
```

Comment:
", run container success ." sounds a bit off - I think this can just be removed.

s/return/print